### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by gadorlhiac**
Landing page for the documentation.

### CODE REVIEW

**Review by fredericpoitevin — APPROVED** [review]
This is great, thanks @gadorlhiac !

What do you think of adding a paragraph on "How to contribute"? In the spirit of it being a living document, we'd like to encourage users to provide constructive feedback, contribute new material, update or correct existing material, etc. Also highlighting the review process as a markedly different approach to what Confluence provides for example.

**Review by apeck12 — APPROVED** [review]
Thanks, this is great!

**Comment by gadorlhiac** [comment]
Great idea @fredericpoitevin for the contribution section. I'll write a section mock-up for review

File: docs/index.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
-    mkdocs.yml    # The configuration file.
-    docs/
-        index.md  # The documentation homepage.
-        ...       # Other markdown pages, images and other files.
+1. 'Before your experiment' pages
+2. 'During your experiment' pages
+3. 'After your experiment' pages
+
+While the documentation can be read in any order, this organization is intended to better highlight pieces of information relevant to each of these stages of the experimental process. The first category includes more in depth information about the theory behind various tools (both hardware and software) available at LCLS. Logistical considerations, such as how to setup SLAC Unix computing accounts, are also covered. The second category of pages is meant to serve as a resource during the beamtime itself. Checklists and step-by-step guides are favored to provide concise, useful reminders which can be read quickly during the experiment. The final category of pages is more analysis oriented and should be a helpful resource when reprocessing data. This again includes explanations of the tools and algorithms, as well as tips and tricks from the accumulated experience of LCLS users and staff.
```
[the remark this answers is not in the document]
Comment (gadorlhiac) [reply docs/index.md:15]: I'll try organizing it similar to that and we can compare

**Pull request merged**